src/include/pgmoneta.h

```diff
@@ -130,6 +130,32 @@ extern "C" {
 #define FORMAT_TEXT           1
 #define BULLET_POINT          "- "
 
+#define COMMAND_BACKUP "backup"
+#define COMMAND_LIST_BACKUP "list-backup"
+#define COMMAND_RESTORE "restore"
+#define COMMAND_VERIFY "verify"
+#define COMMAND_ARCHIVE "archive"
+#define COMMAND_DELETE "delete"
+#define COMMAND_RETAIN "retain"
+#define COMMAND_RESET "reset"
+#define COMMAND_RELOAD "reload"
+#define COMMAND_EXPUNGE "expunge"
+#define COMMAND_ENCRYPT "encrypt"
+#define COMMAND_DECRYPT "decrypt"
+#define COMMAND_COMPRESS "compress"
+#define COMMAND_DECOMPRESS "decompress"
+#define COMMAND_PING "ping"
+#define COMMAND_STOP "stop"
+#define COMMAND_STATUS "status"
+#define COMMAND_STATUS_DETAILS "status-details"
+#define COMMAND_CONF "conf"
+#define COMMAND_CLEAR "clear"
+#define COMMAND_INFO "info"
+#define COMMAND_ANNOTATE "annotate"
+
+#define OUTPUT_FORMAT_JSON "json"
+#define OUTPUT_FORMAT_TEXT "text"
+
```

Comment:
Lets move these back to `cli.c`...

But, lets see if we can use `command_table` instead of the `COMMAND_XYZ`